ports/esp32/boards/OLIMEX_ESP32_POE/board.json

```diff
@@ -0,0 +1,24 @@
+{
+    "deploy": [
+        "../deploy.md"
+    ],
+    "docs": "",
+    "features": [
+        "BLE",
+        "WiFi",
+        "MicroSD",
+        "Battery Charging",
+        "Ethernet",
+        "PoE",
+        "Breadboard friendly"
+    ],
+    "id": "esp32",
+    "images": [
+        "ESP32-POE-ISO-1.jpg"
+    ],
+    "mcu": "esp32",
+    "product": "ESP32",
+    "thumbnail": "",
+    "url": "https://www.olimex.com/",
+    "vendor": "Olimex"
```

Comment:
@robert-hh I see some other boards have this as all uppercase, ie OLIMEX.  Which is correct?

Reply:
The vendor calls itself on the `About` page of it's web site both `Olimex Ltd` and `OLIMEX Ltd.`, so I'll change it to OLIMEX for consistency.